rest/route/task.go

```diff
@@ -43,13 +45,43 @@ func (tgh *taskGetHandler) Factory() gimlet.RouteHandler {
 func (tgh *taskGetHandler) Parse(ctx context.Context, r *http.Request) error {
 	tgh.taskID = gimlet.GetVars(r)["task_id"]
 	_, tgh.fetchAllExecutions = r.URL.Query()["fetch_all_executions"]
+	execution := r.URL.Query().Get("execution")
+
+	if execution != "" && tgh.fetchAllExecutions == true {
```

Comment:
you don't need to add `== true` -- this is implied because fetchAllExecutions is a boolean